voltaire_bundler/execution_endpoint.py

```
@@ -293,14 +301,23 @@ async def execute_cron_job(self, is_debug: bool, bundle_interval: int) -> None:
+                # so each cycle is `bundle_interval` end-to-end rather than
+                # `bundle_duration + bundle_interval`. Without this, a tick
+                # whose work exceeds bundle_interval drags every subsequent
+                # tick later — and the monitor re-add loop downstream gets
+                # progressively more userops past the 5s staleness threshold,
+                # compounding the slip.
+                t0 = time.monotonic()
                 try:
                     await self.bundle_manager.send_next_bundle()
                 except (ValidationException, ExecutionException) as excp:
                     logging.exception(excp.message)
                 except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)